include/SFML/Graphics/RenderTarget.hpp

```diff
@@ -256,6 +256,23 @@ class SFML_GRAPHICS_API RenderTarget : NonCopyable
     virtual Vector2u getSize() const = 0;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Activate or deactivate the render target for rendering
+    ///
+    /// This function makes the render target's context current for
+    /// future OpenGL rendering operations (so you shouldn't care
+    /// about it if you're not doing direct OpenGL stuff).
+    /// Only one context can be current in a thread, so if you
+    /// want to draw OpenGL geometry to another render target
+    /// don't forget to activate it again.
+    ///
```

Comment:
For `RenderWindow::setActive` it is specified that the previously active window (if any) will get deactivated. Here no such statement being made leaves room for interpretation. The expected behaviour is probably something obvious for someone with good knowledge of OpenGL, but I'd still recommend adding a similar statement here or, if applicable, warn the user that (s)he should make sure the previous RT was deactivated (not sure if that last statement would make sense though).